Sources/Tracing/Docs.docc/Guides/ImplementATracer.md

```diff
@@ -0,0 +1,186 @@
+# Implement a Tracer
+
+## Overview
+
+This guide is aimed at ``TracerProtocol`` and ``InstrumentProtocol`` implementation authors.
+
+This guide is for you, if you find yourself in need of implementing your own tracing client such as Zipkin, Jaeger, X-Trace, OpenTelemetry or something similar that is custom to your company or distributed system.
+
+## Do you need an Instrument or a Tracer?
+
+Distributed tracing offers two types of instrumentation protocols: an instrument, and a tracer.
+
+A tracer is-an instrument as well, and further refines it with the ability to start a trace ``Span``.
+
+## Creating an instrument
+
+Creating an instrument means adopting the ``Instrument`` protocol (or ``Tracer`` in case you develop a tracer).
+`Instrument` is part of the `Instrumentation` library & `Tracing` contains the ``Tracer`` protocol.
+
+`Instrument` has two requirements:
+
+1. A method to inject values inside a `FIXME!!!` into a generic carrier (e.g. HTTP headers)
```

Comment:
```suggestion
1. A method to inject values into a `FIXME!!!` a generic carrier (e.g. HTTP headers)
```

Reply:
Thanks, I didn't yet rewrite other guides other than the "Instrument your library" FYI